src/deserialization.hpp

```diff
@@ -356,7 +356,7 @@ Status deserializePredictRequest(
             ov::Tensor tensor;
 
             if (requiresProcessing(requestInput)) {
```

Comment:
If you changed getProcessingHint to getPreProcessingHint maybe change also requiresProcessing to requiresPreProcessing

Reply:
done